src/solvers/dgsem_tree/dg_2d_parabolic.jl

```diff
@@ -132,10 +133,10 @@ function transform_variables!(u_transformed, u, mesh::Union{TreeMesh{2}, P4estMe
 end
 
 # This is the version used when calculating the divergence of the viscous fluxes
-function calc_volume_integral!(du, flux_viscous,
+function calc_volume_integral!(du, flux_viscous::Vector{Array{uEltype, 4}},
```

Comment:
Do we need to specialize on `flux_viscous`? Is specializing `mesh` and `equations_parabolic` not sufficient?

Reply:
At this place not, this is some left-over.